### ISSUE TITLE
Ruby 2.7 Warning: Using the last argument as keyword parameters is deprecated in active_record_prepend.rb

### ISSUE THREAD

**Issue by JunichiIto** [newrelic/newrelic-ruby-agent#337]
I got the following warnings in my Heroku environment:

```
/app/vendor/bundle/ruby/2.7.0/gems/newrelic_rpm-6.10.0.364/lib/new_relic/agent/instrumentation/active_record_prepend.rb:16: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/app/vendor/bundle/ruby/2.7.0/gems/activerecord-6.0.3.1/lib/active_record/suppressor.rb:43: warning: The called method `save' is defined here
```

I'd appreciate if you could fix it.

**Comment by rachelleahklein** [comment]
Hi @JunichiIto, thanks for reaching out. We are working on eliminating the appearance of these warnings and will let you know when we have released a fix.

**Comment by tannalynn** [comment]
Hello @JunichiIto. We have recently released 6.12 which should have these warnings fixed. Thank you for bringing it to our attention!

**Comment by JunichiIto** [comment]
@tannalynn Thank you for the fix and letting me know!
